ui/components/app/transaction-detail-item/transaction-detail-item.component.js

```diff
@@ -58,7 +58,12 @@ export default function TransactionDetailItem({
           </Typography>
         )}
 
-        <Typography variant={TYPOGRAPHY.H7} color={COLORS.UI4} align="end">
+        <Typography
+          variant={TYPOGRAPHY.H7}
+          color={COLORS.UI4}
+          align="end"
+          className="transaction-detail-item__row__subText"
```

Comment:
Thoughts on naming this `row-subtext`? I think it's frowned upon in BEM to have a child of a child.

Reply:
I fixed this

Reply:
****